src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Tabs.js

```diff
@@ -1,4 +1,9 @@
 export class Tabs {
+  constructor () {
+    this.initEventListeners()
```

Comment:
Can't the initEventListeners code not come in the constructor?

Reply:
Wouldn't that make the constructor a bit messy?